tests/test_georaster.py

```diff
@@ -1626,23 +1639,6 @@ def test_coords(self) -> None:
             assert yy.min() == pytest.approx(img.bounds.top + hy)
             assert yy.max() == pytest.approx(img.bounds.bottom - hy)
 
-    def test_eq(self) -> None:
```

Comment:
Did you just delete this test? I don't see any replacement, but maybe I missed something...

Reply:
Ok, there was a duplicate in test_equal, right?

Reply:
Yes this was the original version (day 1!), which has been duplicated since and is useless now